docs/papers/joss/paper.md

```diff
@@ -157,7 +158,7 @@ in different forms. Here, the following categorization is employed:
   state of the material.
 - **Behaviours** describe how a material evolves and reacts locally due
   to gradients inside the material. Here, the material reaction is
-  associated with fluxes (or forces) thermodynamically conjugated with
+  associated with fluxes (or forces) thermodynamically conjugate to
```

Comment:
conjugated ?

Reply:
I like conjugate to, as it's more formal than using the participle ...